subsys/bluetooth/mesh/brg_cfg_cli.c

```diff
@@ -17,30 +17,29 @@ static int32_t msg_timeout;
 
 static struct bt_mesh_brg_cfg_cli *cli;
 
-static int subnet_bridge_status(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
-				struct net_buf_simple *buf)
+static int bridge_status(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
```

Comment:
This is called subnet_bridge_status in the spec. See 4.3.11.3, it is easier to link it to the spec then. This is for alle this op functions.

Reply:
Because we are cutting the names in the header files. We can do that here too.